### DISCUSSION TITLE
How to store data user_id when use createOptionForm ??

### DISCUSSION THREAD

**Discussion by adamyulian** [filamentphp/filament#10492] [Help] [answered]
I have difficulties when using createOptionForm to store user_id from Auth::User()->id. Already try to use methods like mutateFormDataUsing inside the TextInput::make('user_id')->, and try another dumb methods including  like default, defaultvalue, value and still don't work. User_id can't be stored.

**Comment by adamyulian** [comment]
I already get the answer by manually records in model use creating method. 😭😭😭

**Reply by leandrocfe** [reply]
If it works, share the code and mark as answered please.

**Reply by adamyulian** [reply, marked as answer]
```php
protected static function booted() {
        static::creating(function($model) {
            $model->user_id = Auth::user()->id;
        });
    }
``` 

Just use this on your model, and it will always store id user login to user_id in your model.

**Comment by kombiahrk** [comment]
i am doing this by 
creating Forms\Components\Hidden::make('team_id'),
assigning the value by using 
in any one of required field
->afterStateUpdated(function ($state, Forms\Set $set) {
         $set('team_id', Filament::getTenant()->id);
 }),
